src/mbgl/renderer/layers/render_circle_layer.cpp

```diff
@@ -100,9 +100,7 @@ void RenderCircleLayer::render(PaintParameters& parameters, RenderSource*) {
             *parameters.renderPass,
             gfx::Triangles(),
             parameters.depthModeForSublayer(0, gfx::DepthMaskType::ReadOnly),
-            parameters.mapMode != MapMode::Continuous
-                ? parameters.stencilModeForClipping(tile.clip)
-                : gfx::StencilMode::disabled(),
+            gfx::StencilMode::disabled(),
```

Comment:
I was trying to find out why we use stencil clipping in still image mode but couldn't find a good answer. I also don't think there's a valid reason for using clipping in still mode, but not in continuous mode, since it would produce a rendering error in the latter case if we'd wouldn't clip. Our render tests also still continue to work correctly eve with clipping disabled for these layers.